cores.json

```diff
@@ -0,0 +1,1016 @@
+[
+    {
+        "name": "mame2003",
+        "extensions": [
+            "zip"
+        ],
+        "makeoptions": {
+            "buildpath": "./",
+            "makescript": "Makefile",
+            "arguments": []
+        },
+        "options": {
+            "file": "MAME 2003 (0.78)/MAME 2003 (0.78).opt",
+            "settings": {
+                "mame2003_skip_disclaimer": "enabled",
+                "mame2003_skip_warnings": "enabled"
+            }
+        },
+        "license": "LICENSE.md",
+        "repo": "https://github.com/EmulatorJS/mame2003-libretro"
+    },
+    {
+        "name": "vice_x64",
+        "extensions": [
```

Comment:
Could we collapse these long blobs of file extensions into one line

Reply:
VScode decided to help me by cleaning up... 

All cleaned up now.